README.md

```diff
@@ -7,11 +7,26 @@ A language server implementation based on the [Language Server Protocol](https:/
 
 This project contains:
 
-- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 - [MicroProfile Language Server](./microprofile.ls)
+- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 
 You can build all projects at once by running the `buildAll.sh` script (`buildAll.bat` on Windows).
 
+The language features provided by the MicroProfile language server depend on the user's Java project.

```

Comment:
I think this should be changed from `MicroProfile language server` ->  `MicroProfile Language Server` since we use the capitalized version everywhere else